examples/browser-esmodules/example.js

```diff
@@ -0,0 +1,34 @@
+import {v1 as uuidv1, v4 as uuidv4, v3 as uuidv3, v5 as uuidv5} from './node_modules/uuid/esm-browser/index.js';
+
+console.log('uuidv1: ', uuidv1());
+
+
```

Comment:
I would even consider to add `prettier` to get rid of any formatting discussion. However since this PR is already getting really big I wanted to push these kinds of tooling changes and refactorings to a different PR. Fine for you?